.github/workflows/deployments.yml

```diff
@@ -0,0 +1,36 @@
+name: deployments
```

Comment:
nit: do we want this to read as "Deployments" since most of our other tests start with a capitalization of the test group name?

Reply:
sg